README.md

```diff
@@ -29,20 +25,30 @@ Create a new Pulumi project:
 5. Deploy the stack: `pulumi up`
 6. Follow the instructions printed in the console to complete the deployment.
 
-<!-- TODO
 ## Configuration Variables
 
 The following table lists the configuration variables for both templates:
 
-| Variable | Description | Default Value |
-|----------|-------------|---------------|
+| Variable | Description | Type | Default | Required |
+|----------|-------------|------|---------|----------|
+| accountNum | Equinix Fabric billing account number | `integer` | n/a | yes |
+| projectId | Equinix Fabric project Id | `string` | n/a | yes |
+| notificationEmails | Array of contact emails | `list(string)` | n/a | yes |
+| gcpProject | The default GCP project to manage resources in | `string` | n/a | yes |
+| metro | Two-letter code indicating the metropolitan area for the new resources | `string` | "FR" | no |
+| speedInMbps | Equinx Fabric connection bandwidth in Mbps | `integer` | 100 | no |
+| purchaseOrderNum | Equinix Fabric Purchase order number | `string` | "1234" | no |
+| gcpRegion | Region where the GCP resources resides | `string` | "europe-west3" | no |
 
--->
 
 To add them you can use `pulumi config set` command:
 
 ```sh
 pulumi config set projectId <EQUINIX_FABRIC_PROJECT_ID>
+$ pulumi config set accountNum 1234
```

Comment:
`$` should be used consistently. It is only included in code blocks when the following line is the command's output.

https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output